components/Hero.vue

```diff
@@ -13,7 +13,7 @@
           </p>
           <div class="hero__btns mt-4">
             <nuxt-link
-              href="https://app.zencal.io/u/ankit/acorn-globus"
+              href="https://calendar.app.google/So3CCei31oYjhoM47"

```

Comment:
Also my calendar is tweaked for Formester timing etc. You could create one your profile and use that one.